packages/app/android/app/build.gradle

```diff
@@ -51,15 +51,46 @@ android {
         targetSdkVersion flutter.targetSdkVersion
         versionCode flutterVersionCode.toInteger()
         versionName flutterVersionName
+        manifestPlaceholders = [applicationLabel: "Meli"]
     }
 
     buildTypes {
+        debug {
+            applicationIdSuffix ".debug"
+            debuggable true
+            manifestPlaceholders = [applicationLabel: "Meli (Debug)"]
+        }
+        profile {
+            applicationIdSuffix ".profile"
+            manifestPlaceholders = [applicationLabel: "Meli (Profile)"]
+        }
         release {
             // TODO: Add your own signing config for the release build.
             // Signing with the debug keys for now, so `flutter run --release` works.
             signingConfig signingConfigs.debug
         }
     }
+
+    flavorDimensions "appType"
+
+    productFlavors {
+        normal {
+            dimension "appType"
+        }
+        qa {
+            dimension "appType"
+            applicationIdSuffix ".qa"
+            manifestPlaceholders = [applicationLabel: "Meli (QA)"]
+        }
```

Comment:
happy to use a different name if it's helpful for conveying meaning. basically want something that makes it clear that it's some sort of test build (or non-"production")